jquery/index.html

```diff
@@ -320,7 +329,7 @@
 				<section>
 					<h2>jQuery:</h2>
 					<h3>Create and Store</h3>
-					<p>Pass in any HTML string and jQuery will create it and return it as a collection.</p>
+					<p>Pass in any HTML string and jQuery will create it and return it as a array.</p>
```

Comment:
We had an issue for this filed a while ago, which I can't find atm. Basically, the reporter said that it's not an ordinary JavaScript array but an array-like collection. I don't think the distinction is important enough for this level of learning, so let's get this merged.